shell/platform/android/android_image_generator.cc

```diff
@@ -135,7 +135,6 @@ void AndroidImageGenerator::DoDecodeImage() {
 }
 
 bool AndroidImageGenerator::Register(JNIEnv* env) {
-  FML_DCHECK(g_flutter_jni_class->is_null());
```

Comment:
I think this was trying to check that we didn't try do initialize this twice. Is there a better way to do that? Is there a test that would have caught this?

Reply:
`FML_DCHECK(g_flutter_jni_class == nullptr)` would do that.  Not sure if that check is necessary though.  This function is only called during JNI library loading, but even if it were called twice it should be safe.

To test this we would need infrastructure that runs a real Android environment and can start an app that loads a build of `libflutter.so` that was built with assertions enabled.  I don't think the LUCI builders publish those binaries to cloud storage.